test/cpp/interop/interop_server.cc

```diff
@@ -296,6 +296,7 @@ class TestServiceImpl : public TestService::Service {
     StreamingOutputCallRequest request;
     StreamingOutputCallResponse response;
     bool write_success = true;
+    std::unique_ptr<grpc_core::MutexLock> orca_oob_lock;
```

Comment:
Suggest using `absl::optional<>` instead of `std::unique_ptr<>`.  There's no need for dynamic allocation here.

Reply:
Compiler outputs a warning about releasing the mutex that was not held if `absl::optional` is used here.